src/cdrizzlebox.c

```diff
@@ -803,58 +928,80 @@ do_kernel_square(struct driz_param_t *p) {
             xin[3] = xin[0] = (double)i - dh;
             xin[2] = xin[1] = (double)i + dh;
 
-            for (ii = 0; ii < 4; ++ii) {
-                if (interpolate_point(p, xin[ii], yin[ii], xout + ii,
-                                      yout + ii)) {
-                    goto _miss;
-                }
-            }
-
-            /* Work out the area of the quadrilateral on the output grid.
-               Note that this expression expects the points to be in clockwise
-               order */
+	    /* Assuming we don't need to extrapolate, call a more
+	     * efficient interpolator that takes advantage of the fact
+	     * that pixfrac<1 and that we are using a square grid.
+	     */
+	    if (i > 0 && i < mapsize[0] - 1 && j > 0 && j < mapsize[1] - 1) {
```

Comment:
```suggestion
	    if (i > 0 && i < mapsize[0] - 2 && j > 0 && j < mapsize[1] - 2) {
```

JWST regression tests are failing. My investigation shows that this is the most likely culprit.

Reply:
Good catch, thanks.